atlas/src/views/HomeView.tsx

```diff
@@ -34,13 +46,42 @@ const HomeView: React.FC<RouteComponentProps> = () => {
     <>
       <Hero backgroundImg={backgroundImg} />
       <Container>
-        <VideoGallery title="Newest videos" loading={newestVideosLoading} videos={newestVideos} />
-        <VideoGallery
-          title="Featured videos"
-          loading={featuredVideosLoading}
-          videos={featuredVideosData?.featured_videos}
-        />
-        <ChannelGallery title="Newest channels" loading={newestChannelsLoading} channels={newestChannels} />
+        <ErrorBoundary
```

Comment:
Instead of passing the error down and using `ErrorBoundary` to catch it back, let's just do a switch based on `error` right here

Reply:
I changed this to work without the Error Boundary, let me know your thoughts